paddle/phi/infermeta/binary.cc

```diff
@@ -2510,6 +2510,27 @@ void MvInferMeta(const MetaTensor& x, const MetaTensor& vec, MetaTensor* out) {
   out->share_lod(x);
 }
 
+void PullSparseV2InferMeta(const std::vector<const MetaTensor*>& ids,
+                           const std::vector<const MetaTensor*>& w,
+                           int embeddingdim,
+                           int tableid,
+                           const std::string accessorclass,
+                           const std::string ctrlabelname,
```

Comment:
```suggestion
                           const std::string& accessorclass,
                           const std::string& ctrlabelname,
```
现在的错误应该是由于infermeta的机制中没有 const std::string类型的模版实例，所以必须用const std::string&。

Reply:
收到